frontends/web/src/locales/en/app.json

```diff
@@ -397,6 +397,12 @@
     "upgrade": "Upgrade"
   },
   "buy": {
+    "bitrefill": {
+      "error": {
+        "insufficientFunds": "Insufficient funds to buy gift card."
+      },
+      "transactionNote": "Sent to Bitrefill"
+    },
```

Comment:
This file grew organically, since we started with buy and added sell later, we now have keys like: `buy.exchange.sell` 😅 

Maybe it would make sense to make a new top level "market" or "marketplace"  key? but buy works as well no strong opinion.

Reply:
Definitely, I haven't spent much time on lang keys yet, partly for this reason. Probably a larger refactor then with more structured buy/sell/spend categorization

Reply:
it's very annoying to move keys onces they are translated. So if you want to change it would be better to move the new keys before translators start translating the new keys.